runtime/src/parameters.rs

```diff
@@ -26,9 +27,28 @@ parameter_types! {
   pub const MaxAuthorities: u32 = 32;
 
   // Collective
+  // Note: MaxMembers does not influence the pallet logic, but the worst-case weight estimation.
   pub const AdvisoryCommitteeMaxMembers: u32 = 100;
-  pub const AdvisoryCommitteeMaxProposals: u32 = 64;
-  pub const AdvisoryCommitteeMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
+  pub const AdvisoryCommitteeMaxProposals: u32 = 300;
+  pub const AdvisoryCommitteeMotionDuration: BlockNumber = 3 * BLOCKS_PER_DAY;
+  pub const CouncilMaxMembers: u32 = 100;
+  pub const CouncilMaxProposals: u32 = 100;
+  pub const CouncilMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
+  pub const TechnicalCommitteeMaxMembers: u32 = 100;
+  pub const TechnicalCommitteeMaxProposals: u32 = 64;
+  pub const TechnicalCommitteeMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
+
+  // Democracy
+  pub const LaunchPeriod: BlockNumber = 5 * BLOCKS_PER_DAY;
+    pub const VotingPeriod: BlockNumber = 5 * BLOCKS_PER_DAY;
+    pub const FastTrackVotingPeriod: BlockNumber = 3 * BLOCKS_PER_HOUR;
+    pub const MinimumDeposit: Balance = 100 * BASE;
+    pub const EnactmentPeriod: BlockNumber = 2 * BLOCKS_PER_DAY;
```

Comment:
I am not 100% certain which stake they exactly mean by that, but as I understand it relates too all the parts of the runtime where staking is possible, such that stakers have the chance to remove their stake until the proposal is enacted.
If that is the case, all constraints should be met since the only places in the runtime where staking is possible are:
- `court` pallet, unstaking is immediately possible without a time constraint
- `parachain_staking` pallet, anybody can bond/unbond stake or leave completely within 2 rounds (currently 4 minutes) - this will most likely never exceed 1 day.
- `pallet_democracy` not exactly sure if it can be called staking, but voters and delegators back their votes with funds. In that case it is obviously not desired that they can pull out their funds during `EnactmentPeriod`

On a side note, the current configuration only allows voting once for a proposal and doesn't leave room for changing opinions.

`EnactmentPeriod = 2` was chosen, because:
1. Other great Kusama parachains use that
2. It allows to move forward relatively fast with accepted changes (which imo. is appropriate for a fast-changing and experimental system like ours)

I think eventually we will have two configurations for testnet and mainnet, whereas testnet will be much faster paced than mainnet, but for now I think the given configuration is appropriate for both networks.

Reply:
My understanding was that in case of a malignous decision, those who oppose the decision and fear for the economic implications have a chance to unstake and cash out before the change in enacted. Seems like we are well within that timeframe. 👍